assembl/models/idea.py

```diff
@@ -698,7 +698,8 @@ def get_discussion_id(self):
         return self.discussion_id or self.discussion.id
 
     def get_definition_preview(self):
-        body = self.definition.strip()
+        definition = self.definition or ""
```

Comment:
It just makes sure that `definition` is a string so that I can call `strip()` on it on the next line (whereas it was creating an error before in the case of a None value)